### ISSUE TITLE
Port the code to Python3

### ISSUE THREAD

**Issue by nvs-abhilash** [nvs-abhilash/CorrectMe#16]
Currently the code only works with Python2 (As it was first implemented in Python2 back then). I think it should support Python3 as well, as Python2 would be deprecated soon (and is old :/ ).

**Comment by piyush-jaiswal** [comment]
Yes, we should definitely port to Python3. The '2to3' python program might just be useful here.

**Comment by nvs-abhilash** [comment]
Thanks for the suggestion. Worked very well. Didn't check for optimizations but the code is working fine for now. I have put up a PR #22. Check it out.
